268 rta all docs (#3844)
timescale/docs: use-timescale/hyperfunctions/about-hyperfunctions.md

# About Timescale hyperfunctions

> Timescale hyperfunctions are a specialized set of functions that power data analysis on time series and events. 

Reviewer: ```suggestion
Timescale hyperfunctions are a specialized set of functions that power real-time analytics on time series and events. 
```

IoT devices, IT systems, marketing analytics, user behavior, financial metrics, cryptocurrency - these are only a few examples of domains where 
hyperfunctions can make a huge difference. Hyperfunctions provide you with meaningful, actionable insights in real time.  

$CLOUD_LONG includes all hyperfunctions by default, while self-hosted $TIMESCALE_DB includes a subset of them. For
additional hyperfunctions, install the [Timescale Toolkit][install-toolkit] PostgreSQL extension.

## Available hyperfunctions

Here is a list of all the hyperfunctions provided by Timescale. Hyperfunctions

> with a tick in the `Toolkit` column require an installation of Timescale Toolkit for self-hosted deployment. Hyperfunctions

Reviewer: ```suggestion
with a tick in the `Toolkit` column require an installation of Timescale Toolkit for self-hosted deployments. Hyperfunctions
```

> with a tick in the `Experimental` are still under development.

Reviewer: ```suggestion
with a tick in `Experimental` are still under development.
```
Be nice to change experimental to early access in the page.

Author: Experimental is used across many docs, including in an HTML tag, so it would be inconsistent to only change it here. Might be an improvement for another time.
